### ISSUE TITLE
Accessibility improvements - Form fields

### ISSUE THREAD

**Issue by robinmolen** [open-formulieren/open-forms#4716]
### Thema / Theme

Frontend

### Omschrijving / Description

Part of #4622

This issue addresses the issues regarding form fields issues described in the accessibility rapport.
- [x] 1.3.1
- [x] 1.3.5
- [x] 1.4.11
~2.5.3~

**Comment by robinmolen** [comment]
Most form inputs are now connected (with input element `id` and label element `for` tags) to their labels. The date and datetime inputs currently lack this connection. This should be resolved

Some fields have some accessibility issues:
- The map field cannot easily be connected to its label and error message, seeing how this field lacks an input element. I'm also not sure if/how this field can be used without a cursor.
- The signature input has a similar issue. This field has an input element, but it currently expects an bitmap image. I'm also not sure if/how this field can be used without a cursor.
